andela/ah-robotics-frontend
# #163783763 Setup front-end app on heroku

#### What does this PR do?
Setup the front-end app on Heroku.
#### Description of Task to be completed?
- [x] refactor dependencies on package.json
- [x] create application pipeline on Heroku
- [x] link Heroku with the project repository
#### How should this be manually tested?

1. Use the the url below to acess the front-end application on Heroku

```
https://ah-robotics-frontend-stag-pr-9.herokuapp.com/
```
<img width="1221" alt="screenshot 2019-02-11 at 18 20 41" src="https://user-images.githubusercontent.com/9401930/52572837-cda21880-2e29-11e9-901c-49b4cbf4e491.png">

2. Click on the `add-button` it should increment the count as displayed above.

3. Click on the `Redux DevTools` extension on the Chrome browser or right click on the browser select inspect and then choose the redux tool and it should display the store as shown below. 

> The store will update and it will display the count number

> Ps. If you **do not** have `Redux DevTools` installed, find the instructions listed below on how to install the extension.

<img width="714" alt="screenshot 2019-02-06 at 18 53 29" src="https://user-images.githubusercontent.com/6994982/52353907-863e1580-2a40-11e9-931a-8820e3ab8b5f.png">


#### Additional steps to set up your browser for testing React Apps better.

> These tools are not necessary to run the application, they are just extra tools to help you test and debug React applications quickly

1. Install React DevTools extension for Chrome

> Find the extension [Here](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi)

2. Install Redux extension for Chrome

>Find the extension [Here](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=en)
#### What are the relavant pivot tracker stories?
[#163783763](https://www.pivotaltracker.com/n/projects/2235171/stories/163783763)
